tests/pytests/test_profile.py

```diff
@@ -97,12 +97,12 @@ def testProfileSearch(env):
                     ['Type', 'INTERSECT', 'Counter', 1L, 'Children iterators',
                       ['Type', 'TEXT', 'Term', 'hello', 'Counter', 1L, 'Size', 1L],
                       ['Type', 'INTERSECT', 'Counter', 1L, 'Children iterators',
-                        ['Type', 'TEXT', 'Term', 'hello', 'Counter', 1L, 'Size', 1L]
+                        ['Type', 'TEXT', 'Term', 'hello', 'Counter', 1L, 'Size', 1L],
```

Comment:
How did it worked without it? @ashtul FYI

Reply:
It did not.

Reply:
This fix was pushed while RLtest was already broken